server/services/participants-helper.js

```diff
@@ -76,16 +113,26 @@ const run = async (context) => {
     employerSpecificJoin,
     hiredGlobalJoin,
     siteDistanceJoin,
-    siteJoin,
   } = context;
-  let participants = await table.find(criteria, options);
-  participants = addSiteNameToStatusData(participants, employerSpecificJoin, siteJoin);
-  participants = addDistanceToParticipantFields(participants, siteDistanceJoin);
-  participants = scrubParticipantData(
-    participants,
-    (user.isEmployer || user.isHA) && [employerSpecificJoin, hiredGlobalJoin]
-  );
-  return participants;
+  try {
+    let participants = await table.find(criteria, options);
+    participants = flattenParticipants(participants);
+    participants = addSiteNameToStatusData(participants, employerSpecificJoin);
+    participants = addDistanceToParticipantFields(participants, siteDistanceJoin);
+    participants = scrubParticipantData(
+      participants,
+      (user.isEmployer || user.isHA) && [employerSpecificJoin, hiredGlobalJoin]
+    );
+    return participants;
+  } catch (error) {
+    if (['test', 'local', 'dev'].includes(process.env.APP_ENV)) {
+      const sql = await table.find(criteria, { ...options, build: true });
+      // Logging debugging info for dev/local/test environments
+      console.log('participant-helper:run: sql: ', sql); /* eslint-disable no-console */
+      console.log('participant-helper:run: criteria: ', criteria); /* eslint-disable no-console */
```

Comment:
This still brings up GitHub Actions / format-check -> Unexpected console statement. Might need to tweak the eslint flag.

Reply:
Can we use `logger.info` instead of `console.log`?

Reply:
I will update.